OpenEdXMobile/build.gradle

```diff
@@ -397,23 +397,26 @@ android {
             }
         }
 
-        def facebookAppId = ""
         def fbLoginProtocolScheme = ""
         def facebook = config.get('FACEBOOK')
+        def facebookAppId = ""
+        def fbClientToken = ""
         if (facebook?.get('ENABLED')) {
             facebookAppId = facebook.get('FACEBOOK_APP_ID')
-            if (null == facebookAppId) {
-                throw new GradleException("You must set FACEBOOK_APP_ID if FACEBOOK is enabled")
+            fbClientToken = facebook.get('CLIENT_TOKEN')
+            if (null == facebookAppId || null == fbClientToken) {
+                throw new GradleException("You must set FACEBOOK_APP_ID and CLIENT_TOKEN if FACEBOOK is enabled")
             }
-            fbLoginProtocolScheme = "fb"+facebookAppId
+            fbLoginProtocolScheme = "fb" + facebookAppId
         }
 
         manifestPlaceholders = [supportsRtl          : "true",
                                 firebaseEnabled      : firebaseEnabled ?: false,
                                 fcmEnabled           : fcmEnabled,
                                 branchKey            : branchKey,
-                                facebookAppId        : facebookAppId,
                                 microsoftSignature   : microsoftSignature,
+                                facebookAppId        : facebookAppId,
```

Comment:
No need to store `facebookAppId` because the manifest requires an id with `fb` prefix which is our protocol schema.